tap_klaviyo/schemas/campaigns.json

```diff
@@ -20,9 +20,7 @@
         "num_recipients": {
             "type": "integer"
         },
-        "lists": {
-            "$ref": "lists.json"
-        },
+        "lists": {},
```

Comment:
`lists` and `excluded_lists` are both fields that are not mentioned in the API document but API returns in the response. So, we are keeping the `empty` object datatype as the API documentation does not provide exact information.